playbooks/scenario44_rds_instance.yaml

```diff
@@ -100,19 +118,21 @@
         opentelekomcloud.cloud.rds_instance:
           name: "{{ test_rds_name }}"
           state: present
-          availability_zone: "{{ azs['availability_zones'][0]['name'] }},{{ azs['availability_zones'][1]['name'] }}"
+          # availability_zone: "{{ azs['availability_zones'][0]['name'] }},{{ azs['availability_zones'][1]['name'] }}"
+          availability_zone: "{{ azs['availability_zones'][0]['name'] }}"
```

Comment:
any reason for enforcing one specific AZ instead of random AZ?

Reply:
fixed